modules/nf-core/bedtools/intersect/main.nf

```diff
@@ -10,6 +10,7 @@ process BEDTOOLS_INTERSECT {
     input:
     tuple val(meta), path(intervals1), path(intervals2)
     val extension
+    path chrom_sizes
```

Comment:
Can you add `meta` here since it's reference data?

Reply:
When I run the chrom_sizes with meta the test failed, so I removed the meta

Reply:
You also need to update the test to include a meta map

Reply:
```suggestion
    tuple val(meta2), path(chrom_sizes)
```

Reply:
But when one doesn't want to path any genome, and path empty [] (which is what I did in the test) - it failed with meta.

Reply:
You can still pass an empty list, but as a tuple. See my other suggestions